ipalib/frontend.py

```diff
@@ -470,6 +476,41 @@ def __call__(self, *args, **options):
             self.context.principal = getattr(context, 'principal', None)
             return self.__do_call(*args, **options)
 
+    def __audit_to_journal(self, func, params, result):
+        if getattr(context, 'internal_action', None) != func:
```

Comment:
I don't want to bike shed the context name but these are not all internal actions are they? It's more like the current command and the value is stored so we can differentiate between it and any internal operations executed as part of it.

Reply:
I choose to use `audit_action` in the code I am about to push.